atomic_reactor/tasks/binary_container_build.py

```diff
@@ -182,10 +193,14 @@ def build_container(
             "--memory-swap=0",  # TBD: why is this important?
             "--no-cache",  # make sure the build uses a clean environment
             "--pull-always",  # as above
-            # TBD: --authfile? --tls-verify?
             "--squash",
             *cli_buildargs,
         ]
+        if self._registries_authfile:
+            # TBD: this only works if the OSBS deployment uses a single registry secret
+            # TBD: we also can't properly handle "insecure" config for pull registries
```

Comment:
I don't see an issue with using 1 single .dockerconfigjson with all secrets. I believe for all supported registries in OSBS we should have an entry in there or do I miss some other valid scenario?

yeah, I haven't thought about handling insecure configs. Is it a blocker now?

Reply:
Definitely not a blocker, just something to address in the future :)

Same for the .dockerconfigjson - in our configmap, we happen to use the same one for all the registries. But nothing stops an OSBS admin from using two or more different dockerconfigs.

Both problems are non-blockers for the initial implementation IMO

Reply:
yeah I've filed different story for those multiple possible secrets for registries and to get rid of that